basics/data-import/pinot-file-system/README.md

```diff
@@ -1,17 +1,17 @@
 ---
 description: >-
   This section contains a collection of short guides to show you how to import
-  from a Pinot supported file system.
+  data from a Pinot-supported file system.
 ---
 
 # File Systems
 
-FileSystem is an abstraction provided by Pinot to access data in distributed file systems (DFS).
+FileSystem is an abstraction provided by Pinot to access data stored in distributed file systems (DFS).
 
-Pinot uses distributed file systems for the following purposes:
+Pinot uses distributed file systems for:
 
-* Batch Ingestion Job - To read the input data (CSV, Avro, Thrift, etc.) and to write generated segments to DFS
-* Controller - When a segment is uploaded to the controller, the controller saves it in the DFS configured.
+* Batch Ingestion Job - To read the input data (CSV, Avro, Thrift, etc.) and to write generated segments to DFS.
```

Comment:
```suggestion
* **Batch ingestion jobs**: To read the input data (CSV, Avro, Thrift, etc.) and to write generated segments to DFS.
```

Reply:
re: https://developers.google.com/style/lists#description-lists-that-use-run-in-headings